application/pom.xml

```diff
@@ -20,6 +20,10 @@
             <groupId>de.starwit</groupId>
             <artifactId>persistence</artifactId>
         </dependency>
+        <dependency>
```

Comment:
So we are aiming for a keycloak specific integration?

Reply:
it was just the best-working way...